dxr/plugins/python/indexers.py

```diff
@@ -35,19 +42,115 @@
         'properties': {
             'py_type': NEEDLE,
             'py_function': NEEDLE,
+            'py_derived': NEEDLE,
+            'py_bases': NEEDLE,
         },
     },
 }
 
 
+class TreeToIndex(TreeToIndexBase):
+    def __init__(self, *args, **kwargs):
+        super(TreeToIndex, self).__init__(*args, **kwargs)
+
+        self.python_path = getattr(self.tree, 'plugin_python_path', self.tree.source_folder)
+        self.class_bases = {}
+        self.derived_classes = defaultdict(list)
+        self.names = {}
+
+    @property
+    def unignored_files(self):
```

Comment:
I like this. I like this so much, it's making me think that perhaps build.unignored_files() should become a method on TreeConfig. What do you think?